docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -0,0 +1,871 @@
+[[get-started-elastic-stack]]
+== Getting started with the {stack}
+
+Looking for an {stack} ("ELK tutorial") that shows how to set up the {stack}? In
+this tutorial, you learn how to get up and running quickly. First you install
+the core open source products:
+
+* <<install-elasticsearch,{es}>>
+* <<install-kibana,{kib}>>
+* <<install-beats,{beats}>>
+* <<install-logstash,{ls}>>
+
+Then you learn how to implement a system monitoring solution that uses
+{metricbeat} to collect server metrics and ship the data to {es}, where you can
+search and visualize the data by using {kib}. After you get the basic setup
+working, you add {ls} for additional parsing.
+
+To get started, you can install the {stack} on a single VM or even on your
+laptop. 
+ 
+IMPORTANT: Implementing security is a critical step in setting up the {stack}.
+To get up and running quickly with a sample installation, you skip those steps
+right now, but keep in mind that security is a critical component of your
+deployment.
+
+// REVIEWERS: I haven't discussed security because it really complicates the learning
+// experience. I think there's a value to teaching users how to set up a basic
+// installation (and get the various components talking to each other) before
+// adding security.
+
+[float]
+[[install-prereqs]]
+=== Before you begin
+
+* See the https://www.elastic.co/support/matrix[Elastic Support
+Matrix] for information about supported operating systems and product
+compatibility.
+
+* Verify that your system meets the
+https://www.elastic.co/support/matrix#matrix_jvm[minimum JVM requirements] for
+{ls} and {es}.
+
+// REVIEWERS: In the old version of this doc, we mentioned specific versions and
+// pointed users to the download pages, but I think that might be too much hand
+// holding. IMO, the support matrix should be the place where users go to
+// make sure they meet system requirements. Is there anything else we need to
+// say here about Java requirements here that aren't covered in the matrix?
+
+
+[float]
+[[install-elasticsearch]]
+=== Install {es}
+
+https://www.elastic.co/products/elasticsearch[{es}] is a real-time,
+distributed storage, search, and analytics engine. It can be used for many
+purposes, but one context where it excels is indexing streams of semi-structured
+data, such as logs or decoded network packets.
+
+To download and install {es}, open a terminal window and use the commands that
+work with your system (<<deb, deb>> for Debian/Ubuntu, <<rpm, rpm>> for
+Redhat/Centos/Fedora, <<mac, mac>> for OS X, and <<win, win>> for Windows):
+
+// REVIEWERS: Should we add Linux to this list? If so, how do the commands differ
```

Comment:
@yaauie I've updated the instructions to show the tar.gz instead of the .zip, but I think we need to avoid putting too much installation info in this tutorial. We want to keep the installation steps simple and quick...and show the installation method that's the most common for the platforms  we expect users to try this out on. User should look at the installation docs for info about the available packages and installing in a production environment. Sound fair?